app/test_main.py

```diff
@@ -1,3 +1,26 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (50, 50, [8, 7]),
+        (24, 24, [2, 2]),
+        (23, 25, [1, 2]),
+        (27, 25, [2, 2]),
```

Comment:
The expected result for `get_human_age(27, 25)` should be `[2, 2]`. However, according to the conversion rules, 27 cat years should convert to 2 human years, and 25 dog years should convert to 2 human years. Please ensure the conversion logic aligns with the task description.